scripts/evergreen_api_server

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
to provide compatibility between the old programs and the new binary, so that we can decouple the ops work (and make potential rollbacks across this binary possible.